community/cypher/compatibility-suite/src/test/resources/cypher/db/config/cost-compiled.json

```diff
@@ -1,4 +1,4 @@
 {
   "planner" : "cost",
-  "runtime" : "compiled"
+  "runtime" : "interpreted"
```

Comment:
This file may be removed completely, since you're removing its only usage above.